tests/test_js_functions.py

```diff
@@ -57,6 +57,38 @@ def test_createElement():
     document.createElement();
     """)
 
+def test_enablePrivilege():
+    """Tests that enablePrivilege throws a warning."""
+
+    err = _do_test_raw("""
+        netscape.security.PrivilegeManager.enablePrivilege()
+    """)
+    assert err.warnings
+
+    eq_(err.warnings[0]["id"],
+        ("js", "traverser", "dangerous_global"))
+    eq_(err.warnings[0].get("signing_severity"), "high")
+
+def test_privileged_unprivileged_interacion():
+    """Tests that functions which may lead to privilege escalation are
+    detected."""
+
+    for meth in "cloneInto", "exportFunction":
+        err = _do_test_raw("""
+            Cu.%s(foo)
+        """ % meth)
+        assert err.warnings
+
+        eq_(err.warnings[0]["id"],
+            ("js", "traverser", "dangerous_global"))
+        eq_(err.warnings[0].get("signing_severity"), "low")
```

Comment:
On Tue, May 05, 2015 at 07:10:11AM -0700, Mathieu Agopian wrote:
>Did you mean to make it a `low` signing severity?

Yes. I think this is relatively benign, but I could be convinced 
otherwise.